core/Measurable/Type/Manager.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+namespace Piwik\Measurable\Type;
+
+use Piwik\Plugin\Manager as PluginManager;
+use Piwik\Measurable\Type;
+
+class Manager
```

Comment:
If we could avoid another `Manager` class that would be great :)

`use Piwik\Plugin\Manager as PluginManager` are so boring to write (e.g. in this very file), and `Manager` type-hints are confusing. `TypeManager` or `MeasurableTypeManager`? (I have no preference between both)